region/wreckedship/main/Bowling Alley.json

```diff
@@ -791,6 +890,77 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 3],
+      "name": "Leave Shinecharged (X-Mode, Bowling)",
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"doorUnlockedAtNode": 2},
+        "canXMode",
+        "h_XModeSpikeHit",
+        "h_shinechargeMaxRunway",
+        "canShinechargeMovement",
+        {"shineChargeFrames": 145}
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 3],
+      "name": "Leave with Shinespark (X-Mode, Bowling)",
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"doorUnlockedAtNode": 2},
+        "canXMode",
+        "h_XModeSpikeHit",
+        "h_shinechargeMaxRunway",
+        "canShinechargeMovement",
+        {"shineChargeFrames": 145},
+        {"shinespark": {"frames": 2}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark":{}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 3],
+      "name": "Leave Shinecharged (Speedkeep Shinecharge, Bowling)",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 19
+        }
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
+        "canSpeedKeep",
+        "canCarefulJump",
```

Comment:
```suggestion
```
Generally low-level movement tech like "canCarefulJump" is irrelevant in strats with much higher difficulty like this one.